pkgs/development/libraries/gnutls/default.nix

```diff
@@ -64,7 +64,9 @@ stdenv.mkDerivation rec {
     hash = "sha256-90/FlUsn1Oxt+7Ed6ph4iLWxJCiaNwOvytoO5SD0Fz4=";
   };
 
-  outputs = [ "bin" "dev" "out" "man" "devdoc" ];
+  outputs = [ "bin" "dev" "out" ]
+    ++ lib.optionals (!stdenv.hostPlatform.isMinGW) [ "devdoc" "man" ];
```

Comment:
Keeping the same order of "devdoc" and "man" should avoid mass-rebuild.

Reply:
(otherwise we can't merge directly to master)

Reply:
My bad. Fixed